pandas/plotting/_core.py

```diff
@@ -2156,10 +2156,18 @@ def hist_frame(data, column=None, by=None, grid=True, xlabelsize=None,
         The size of the figure to create in inches by default
     layout : tuple, optional
         Tuple of (rows, columns) for the layout of the histograms
-    bins : integer, default 10
-        Number of histogram bins to be used
+    bins : integer or sequence, default 10
+        Number of histogram bins to be used. If an integer is given, bins + 1
+        bin edges are calculated and returned. If bins is a sequence, gives
+        bin edges, including left edge of first bin and right edge of last
+        bin. In this case, bins is returned unmodified.
     `**kwds` : other plotting keyword arguments
         To be passed to hist function
+
+    Notes
+    -----
+    See matplotlib documentation online for more on this
```

Comment:
Could you point to a specific `matplotlib` function(s) with a "See Also" section in the docstring?

Reply:
`matplotlib.axes.Axes.hist` *might* be turned into a link automatically, if you add it to "See Also".